### ISSUE TITLE
Default style for alignment

### ISSUE THREAD

**Issue by rutiolma** [openfed/kiso#8]
It is important to have default styles for content created using ckeditor. 
Ckeditor/Drupal render content in a specific way and create classes based on the selected options. For example, when adding an image on a body field with ckeditor, the image plugin allows the content editor to define the position of the image (left/center/right) and add a caption. The output generated is something like:
```
<figure role="group" class="align-center">
  <img alt="test" data-entity-type="image" data-entity-uuid="0" height="322" src="[IMAGE]" width="322">
  <figcaption>Brief description</figcaption>
</figure>
```
In this case, there is no style for "figure.align-center" (same for left/right) and neither for "figure.figcaption".

I guess that Kiso should at least have predefined styles for the basic ckeditor plugins on Openfed, if that's not the case already (ex: image, links, position, quotes, bullets...)

**Comment by smillart** [comment]
Each HTML elements have default styles anywhere. A `.text-formatted` class is added on the `div` wrappers of CKEditor fields to assist with predefined styling of base elements and those with classes assigned to them:

- https://github.com/openfed/kiso/blob/8.2.x-dev/templates/field/field--text.html.twig
- https://github.com/openfed/kiso/blob/8.2.x-dev/templates/field/field--text-long.html.twig
- https://github.com/openfed/kiso/blob/8.2.x-dev/templates/field/field--text-with-summary.html.twig
- https://github.com/openfed/kiso/blob/8.2.x-dev/scss/components/field/text-formatted.scss

However, I noticed that only `img` (image) elements have predefined styles when floated (left/right) or centered. I gonna solve this issue in the next Kiso release regarding other elements like figures, videos, blockquotes, etc.

**Comment by smillart** [comment]
Default styles for alignment will be available for the next [openfed/kiso 2.5] release.
